server/dive_tasks/tasks.py

```diff
@@ -515,12 +516,12 @@ def convert_images(self: Task, folderId):
     manager: JobManager = self.job_manager
 
     items = gc.listItem(folderId)
-    skip_item = (
-        lambda item: item["name"].endswith(".png")
-        or item["name"].endswith(".jpeg")
-        or item["name"].endswith(".jpg")
-    )
-    items_to_convert = [item for item in items if not skip_item(item)]
+    # Start here
+    items_to_convert = [
+        item
+        for item in items
+        if (imageRegex.match(item["name"]) and not safeImageRegex.match(item["name"]))
```

Comment:
This filter isn't function properly.  My mix of png and tiff aren't converting anything.

Reply:
s/match/search/g